src/Pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Layout, Menu, Icon } from 'antd';
+import { AdminPageHeader } from '../../components/AdminPageHeader/AdminPageHeader';
+
+const { Content, Footer, Sider } = Layout;
+const { SubMenu } = Menu;
+
+export const AdminDashboard = () => {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const toggle = (): void => {
+    setCollapsed(!collapsed);
+  };
+
+  return (
+    <Layout style={{ minHeight: '100vh' }}>
+      <Sider trigger={null} collapsible collapsed={collapsed}>
+        <div className="logo" />
+        <Menu theme="dark" defaultSelectedKeys={['1']} mode="inline">
+          <Menu.Item key="1">
+            <Icon type="pie-chart" />
+            <span>Option 1</span>
+          </Menu.Item>
+          <Menu.Item key="2">
+            <Icon type="desktop" />
+            <span>Option 2</span>
+          </Menu.Item>
+          <SubMenu
+            key="sub1"
+            title={
+              <span>
+                <Icon type="user" />
+                <span>User</span>
+              </span>
+            }
+          >
+            <Menu.Item key="3">Tom</Menu.Item>
+            <Menu.Item key="4">Bill</Menu.Item>
+            <Menu.Item key="5">Alex</Menu.Item>
+          </SubMenu>
+          <SubMenu
+            key="sub2"
+            title={
+              <span>
+                <Icon type="team" />
+                <span>Team</span>
+              </span>
+            }
+          >
+            <Menu.Item key="6">Team 1</Menu.Item>
+            <Menu.Item key="8">Team 2</Menu.Item>
+          </SubMenu>
+          <Menu.Item key="9">
+            <Icon type="file" />
+            <span>File</span>
+          </Menu.Item>
+        </Menu>
+      </Sider>
+      <Layout>
+        <AdminPageHeader toggle={toggle} collapsed={collapsed} />
+        <Content style={{ margin: '24px', backgroundColor: '#fff' }}>
```

Comment:
The main content can be passed to template by children prop or `mainContent` prop.